velox/exec/RowContainer.h

```diff
@@ -506,6 +562,35 @@ class RowContainer {
     }
   }
 
+  template <typename T>
+  static void extractValuesOffsetsWithNulls(
+      const char* const* rows,
+      const BufferPtr& offsets,
+      int32_t columnOffset,
+      int32_t nullByte,
+      uint8_t nullMask,
+      const vector_size_t resultOffset,
+      FlatVector<T>* result) {
+    auto numRows = offsets->size() / sizeof(vector_size_t);
+    auto offsetsVector = offsets->as<vector_size_t>();
```

Comment:
auto* rawOffsets for consistency with the rest of the code base

(auto* rawRowNumbers once we rename offsets to rowNumbers)

Reply:
I was trying to be consistent with the rest of the code in this file. Since there is no usage of the word 'raw' in this file.

But I agree the rest of the code uses that naming. wdyt ?